src/lib/common/clockFunctions.cpp

```diff
@@ -50,7 +50,7 @@ void clock_difftime(struct timespec* endTime, struct timespec* startTime, struct
 *
 * clock_addtime - 
 */
-void clock_addtime(struct timespec* accTime, struct timespec* diffTime)
+void clock_addtime(struct timespec* accTime, const struct timespec* diffTime)
```

Comment:
Let's not mix C-pointers and C++ references, the call will look really ugly.
It is true a change like this would somehow add to the understanding of the function, but still.
I would keep it as is. IMO this extra 'const' is enough to show which is input/output.